server/routes/prisonerReport/index.ts

```diff
@@ -0,0 +1,28 @@
+import express, { RequestHandler, Router } from 'express'
+import asyncMiddleware from '../../middleware/asyncMiddleware'
+
+import PrisonerReportRoutes from './prisonerReport'
+
+import PlaceOnReportService from '../../services/placeOnReportService'
+import ReportedAdjudicationsService from '../../services/reportedAdjudicationsService'
+import LocationService from '../../services/locationService'
+
+export default function prisonerReportRoutes({
+  reportedAdjudicationsService,
+  placeOnReportService,
+  locationService,
+}: {
+  reportedAdjudicationsService: ReportedAdjudicationsService
+  placeOnReportService: PlaceOnReportService
+  locationService: LocationService
+}): Router {
+  const router = express.Router()
+
+  const prisonerReport = new PrisonerReportRoutes(reportedAdjudicationsService, placeOnReportService, locationService)
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+
+  get('/:prisonerNumber/:adjudicationNumber', prisonerReport.view)
```

Comment:
I assume that we actually only logically need the adjudication number at the prisonerNumber is just there to make things nicer in the url

Reply:
I use the prisonerNumber from the params in the controller as we need to call prisonAPI to get the prisoner details for the header thingy (location, name etc)

Reply:
Yeah, i am not sure about that, because we have to get the adjudication anyway, and that contains the prisoner number.

But I don't mind that too much ATM